xarray/plot/plot.py

```diff
@@ -83,8 +83,32 @@ def _easy_facetgrid(darray, plotfunc, x, y, row=None, col=None,
     return g.map_dataarray(plotfunc, x, y, **kwargs)
 
 
-def plot(darray, row=None, col=None, col_wrap=None, ax=None, rtol=0.01,
-         subplot_kws=None, **kwargs):
+def _line_facetgrid(darray, row=None, col=None, hue=None,
+                    col_wrap=None, sharex=True, sharey=True, aspect=None,
+                    size=None, subplot_kws=None, **kwargs):
+    """
+    Convenience method to call xarray.plot.FacetGrid for line plots
+    kwargs are the arguments to pyplot.plot()
+    """
+    ax = kwargs.pop('ax', None)
+    figsize = kwargs.pop('figsize', None)
+    if ax is not None:
+        raise ValueError("Can't use axes when making faceted plots.")
+    if aspect is None:
```

Comment:
Use `aspect=1`, `size=3` as default arguments in the `def _line_facetgrid` line?

Reply:
It's intentional that you can't pass `figsize` to facetted plots, because it's ambiguous whether you are referring to the size of each panel or the size of the entire plot. The later is what `figsize` means in matplotlib.

Reply:
If I change the signature of `_easy_factgrid` to have `aspect=1` and `size=3`  and remove the lines 

````python
if aspect is None:
    aspect = 1
if size is None:
    size = 3
````
then some tests fail, presumably because some functions explicitly pass `None` as arguments. I suggest that for consistency we leave the signature of `_line_facetgrid` similar to `_easy_facetgrid`, but if you think it's not important then I can change it.

Reply:
Best to leave it as it is then.